File: app/src/main/java/edu/application/data/data_sources/room/entites/LessonEntity.java
package edu.application.data.data_sources.room.entites;

import androidx.room.ColumnInfo;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

import edu.application.data.models.Lesson;

@Entity
public class LessonEntity {
    @PrimaryKey(autoGenerate = true)
    public int id;

    @ColumnInfo
    public String textName, textAuthor, text, size;

    @ColumnInfo
    public int ageMin;

    @ColumnInfo
    public String questions;

    public LessonEntity(String textName, String textAuthor, String text, String size, int ageMin,
                        String questions) {
        this.textName = textName;
        this.textAuthor = textAuthor;
        this.text = text;
        this.size = size;
        this.ageMin = ageMin;
        this.questions = questions;
    }
}

File: app/src/main/java/com/raz0ne/speedreading_training/data/data_sources/room/entites/LessonEntity.kt
package com.raz0ne.speedreading_training.data.data_sources.room.entites

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "lessons")
class LessonEntity(

    @ColumnInfo
    var textName: String,

    @ColumnInfo
    var textAuthor: String,

    @ColumnInfo
    var text: String,

    @ColumnInfo
    var size: String,

    @ColumnInfo
    var ageMin: Int,

    @ColumnInfo
    var questions: String
) {
    @PrimaryKey(autoGenerate = true)
    @ColumnInfo
    var id = 0
}